Sources/Networking/HTTPClient/NetworkError.swift

```diff
@@ -175,6 +175,38 @@ extension NetworkError: PurchasesErrorConvertible {
 
 extension NetworkError: DescribableError {}
 
+extension NetworkError: LocalizedError {
+
+    var errorDescription: String? {
```

Comment:
This was also a big part of this PR, we'll stop showing "The operation couldn’t be completed." whenever we log a `NetworkError`.